### PR TITLE
Feat/a11y

### PR DESCRIPTION
**Pull request by trinegl**

### CODE REVIEW

File: _includes/work.liquid [hunk 1]
```diff
@@ -29,34 +33,34 @@
...
-					<img src="./assets/Comdirect.svg" alt="" />
+					<img src="./assets/Comdirect.svg" alt="Comdirect logo" />
 				</div>
 				<div class="work__logos-el-wrapper">
-					<img src="./assets/ADAC.svg" alt="" />
+					<img src="./assets/ADAC.svg" alt="ADAC logo" />
 				</div>
 				<div class="work__logos-el-wrapper">
-					<img src="./assets/Telefonica.svg" alt="" />
+					<img src="./assets/Telefonica.svg" alt="Telefonica logo" />
```
Comment (timbru31) [inline _includes/work.liquid:48]: I think we should use the correct brand name here:

```suggestion
					<img src="./assets/Telefonica.svg" alt="Telefónica logo" />
```

File: _includes/work.liquid [hunk 2]
```diff
@@ -29,34 +33,34 @@
...
-					<img src="./assets/VW.svg" alt="" />
+					<img src="./assets/VW.svg" alt="Volkswagen logo" />
 				</div>
 				<div class="work__logos-el-wrapper">
-					<img src="./assets/Audi.svg" alt="" />
+					<img src="./assets/Audi.svg" alt="Audi logo" />
 				</div>
 				<div class="work__logos-el-wrapper">
-					<img src="./assets/Comdirect.svg" alt="" />
+					<img src="./assets/Comdirect.svg" alt="Comdirect logo" />
```
Comment (timbru31) [inline _includes/work.liquid:42]: Tricky a11y wise, but the brand is styled as `comdirect`

**Review by timbru31 — COMMENTED** [review]
Wow, that's awesome - thanks. Two minor adjustments

File: _includes/footer.liquid [hunk 3]
```diff
@@ -4,13 +4,13 @@
...
                           col-start-5-xs col-end-12-xs">
-        <img alt="Accenture Interactive" class="accenture-brand__logo" src="/assets/svg/accenture-logo.svg">
+        <img alt="text: part of Accenture Interactive" class="accenture-brand__logo" src="/assets/svg/accenture-logo.svg">
       </div>
     </div>
   </div>
   <div class="grid footer__wrapper">
     <a class="footer__s2-link col-start-1 col-end-6" href="/">
-      <img alt="Sinnerschrader" src="/assets/svg/sinnerschrader-logo.svg"/>
+      <img alt="Sinnerschrader logo" src="/assets/svg/sinnerschrader-logo.svg"/>
```
Comment (timbru31) [inline _includes/footer.liquid:13]:
```suggestion
      <img alt="SinnerSchrader logo" src="/assets/svg/sinnerschrader-logo.svg"/>
```

**Review by timbru31 — APPROVED** [review]

**Review by Wandkleister — APPROVED** [review]

**Review by tam-kis — APPROVED** [review]

**Pull request merged**